arviz/tests/base_tests/test_plots_bokeh.py

```diff
@@ -1104,3 +1105,110 @@ def test_plot_bpv_discrete():
         show=False,
     )
     assert axes.shape
+
+
+@pytest.mark.parametrize(
+    "kwargs",
+    [
+        {},
+        {"x": "x1"},
+        {"x": ("x1", "x2")},
+        {
+            "x": ("x1", "x2"),
+            "y_kwargs": {"fill_color": "blue"},
+            "y_hat_plot_kwargs": {"fill_color": "orange"},
+            "legend": True,
+        },
+        {"x": ("x1", "x2"), "y_model_plot_kwargs": {"line_color": "red"}},
+        {
+            "x": ("x1", "x2"),
+            "kind_pp": "hdi",
+            "kind_model": "hdi",
+            "y_model_fill_kwargs": {"color": "red"},
+            "y_hat_fill_kwargs": {"color": "cyan"},
+        },
+    ],
+)
+def test_plot_lm(models, kwargs):
+    idata = models.model_1
+    if "constant_data" not in idata.groups():
+        y = idata.observed_data["y"]
+        x1data = y.coords[y.dims[0]]
+        idata.add_groups({"constant_data": {"_": x1data}})
+        idata.constant_data["x1"] = x1data
+        idata.constant_data["x2"] = x1data
+
+    axes = plot_lm(
+        idata=idata, y="y", y_model="eta", backend="bokeh", xjitter=True, show=False, **kwargs
+    )
+    assert np.all(axes)
+
+
+def test_plot_lm_multidim(multidim_models):
+    idata = multidim_models.model_1
+    axes = plot_lm(idata=idata, y="y", plot_dim="dim1", show=False, backend="bokeh")
+    assert np.any(axes)
+
+
+def test_plot_lm_list():
+    y = [1, 2, 3, 4, 5]
+    assert plot_lm(y=y, x=np.arange(len(y)), show=False, backend="bokeh")
+
+
+def test_plot_lm_bad_kind(models, multidim_models):
+    idata1 = models.model_1
+    idata2 = multidim_models.model_1
+    with pytest.raises(ValueError):
```

Comment:
Can this test be parameterized?

Reply:
How about adding 3 different methods for testing ValueError, TypeError and UserWarning?

Reply:
I have tried to make these tests patrameterized.

Reply:
Theres still many things being tested in the same test. Can we split these tests apart so each `pytest.raises` is in its own test? Or is this not possible?

Reply:
Okay! Got it! making the changes..